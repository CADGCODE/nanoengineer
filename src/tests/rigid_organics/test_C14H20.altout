======= test_C14H20.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= stderr =======
cutover to fine tolerance at 3
== structure comparison ==
error reading atom 0 from test_C14H20.xyzcmp
could not read base positions file from -B<filename>
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_C14H20.mmp  > test_C14H20.xyz
final RMS gradient=2213.555271 after 6 iterations
== structure comparison ==
======= test_C14H20.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_C14H20.mmp
# Output File: test_C14H20.xyz
# Trace File: test_C14H20.trc
# Number of Atoms: 34
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#
0 3049.01 4454.27
1 2289.11 2739.84
2 2218.61 2628.97
3 2213.86 2621.64
4 2213.57 2621.19
5 2213.56 2621.16
6 2213.56 2621.16
# Done: Minimization final rms: 2213.555271, highForce: 2621.160475
======= test_C14H20.xyz =======
34
RMS=2213.555271
C 0.263451 -0.507694 -1.366625
C -0.365063 0.891423 -1.131418
C -0.143931 -1.463308 -0.210654
C 1.819032 -0.375648 -1.413514
C 0.144015 1.464561 0.215420
C -1.919107 0.764575 -1.085853
C -1.698821 -1.589559 -0.167328
C 0.365748 -0.889624 1.133838
C 1.698618 1.590916 0.170191
C -0.263667 0.508114 1.368768
C 1.919742 -0.762765 1.086946
C -1.819113 0.376954 1.413896
C -2.336516 -0.190558 0.063655
C 2.336534 0.192341 -0.062985
H -0.081305 -0.909291 -2.317619
H -0.084492 1.563204 -1.942120
H 0.284785 -2.450557 -0.380270
H 2.267534 -1.361918 -1.592199
H 2.107559 0.297378 -2.232812
H -0.285065 2.451255 0.381976
H -2.364527 1.755019 -0.917799
H -2.284200 0.363146 -2.040872
H -1.990466 -2.272216 0.639198
H -2.059990 -2.003684 -1.119086
H 0.084632 -1.562054 1.943268
H 2.061089 2.005484 1.120428
H 1.990594 2.275033 -0.632777
H 0.081184 0.910070 2.318804
H 2.284255 -0.361280 2.042631
H 2.364528 -1.753751 0.917915
H -2.107630 -0.296330 2.232824
H -2.267421 1.363673 1.592210
H -3.423084 -0.276941 0.085565
H 3.423098 0.278034 -0.084601
