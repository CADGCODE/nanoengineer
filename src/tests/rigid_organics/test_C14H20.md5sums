stdout 7B373C2EBEAA7CBDBBF1F23C2084FAD1
stderr D41D8CD98F00B204E9800998ECF8427E
test_C14H20.xyz 1B8EE294B6D74A42DD36EF8567ED5A63
