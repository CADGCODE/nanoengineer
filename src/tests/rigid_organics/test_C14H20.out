======= test_C14H20.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= stderr =======
cutover to fine tolerance at 3
== structure comparison ==
error reading atom 0 from test_C14H20.xyzcmp
could not read base positions file from -B<filename>
